packages/ui/package.json

```diff
@@ -108,7 +108,7 @@
     "lint": "eslint \"src/**/*.{ts,tsx}\"",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook -o .storybook/dist",
-    "test": "jest src",
+    "test": "NODE_ENV=test jest ui -i",
```

Comment:
Is `-i` needed? Doesn't it just make tests run serially instead of in parallel?

Reply:
true, tried the same command as in the server